.github/ISSUE_TEMPLATE/add-api-user.yml

```diff
@@ -1,4 +1,4 @@
-name: EddieHub API
+name: EddieHub API add
```

Comment:
I would suggest this 🤔 

```suggestion
name: Get Added to the EddieHub API
```

__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix